tests/qos/files/qos.yml

```diff
@@ -2987,8 +2987,8 @@ qos_params:
                     ecn: 1
                     pgs: [0, 0, 0, 0, 0, 3, 4, 3, 4, 3, 4, 3, 4, 3, 4, 3, 4]
                     queues: [0, 0, 0, 0, 0, 3, 4, 3, 4, 3, 4, 3, 4, 3, 4, 3, 4]
-                    src_port_ids: [0, 4, 16, 20, 34, 0, 0, 4, 4, 16, 16, 20, 20, 34, 34, 36, 36]
-                    dst_port_ids: [37, 38, 39, 42, 44, 37, 37, 38, 38, 39, 39, 42, 42, 44, 44, 45, 45]
+                    src_port_ids: [0, 1, 2, 3, 4, 0, 0, 1, 1, 2, 2, 3, 3, 4, 4, 5, 5]
```

Comment:
Why are these changed? These were the configs for 100G

Reply:
The test has been uplifted as part of the new platform support for 400G. It is now agnostic across port layouts. Instead of these IDs being unique port IDs, they're now indices that the test parses into whatever valid ports the testbed has. Thus the 100G test needed to receive this config change as well.

Reply:
I went ahead and made an internal preview of the 202012 branch double commit for these tests, and have confirmed that these changes do not affect the tests.